dev-tools/action-install-minikube.sh

```diff
@@ -24,26 +29,55 @@ if [[ "$OSTYPE" == "linux-gnu"* ]]; then
         # Update permissions for .kube and .minikube
         sudo chown -R $USER $HOME/.kube $HOME/.minikube
 
+        export IP=$(minikube ip)
+        sudo sed -i'' '/minikubeCA/d' /etc/hosts
+        echo "$IP minikubeCA" | sudo tee -a  /etc/hosts
+        cd ~
+        kubectl config set-cluster minikube --server=https://minikubeCA:8443 --certificate-authority=.minikube/ca.crt
+        kubectl config set-context minikube --user=minikube --cluster=minikube
+        kubectl config use-context minikube
+
         # Run Minikube Tunnel
         sudo nohup minikube tunnel >/tmp/tunnel.out 2>/tmp/tunnel.out &
 
 elif [[ "$OSTYPE" == "darwin"* ]]; then
         # Mac OSX
         PLATFORM=darwin-amd64
-        echo "unsupported operating system; ignoring fluvio install"
+        
+        curl -LO https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-${PLATFORM}
+        sudo install minikube-${PLATFORM} /usr/local/bin/minikube
+
+        # Install hyperkit using brew for osx minikube driver
+        brew install hyperkit
+
+        # Start Minikube with `hyperkit` driver
+        # OSx does not have `none` bare metal option
+        minikube start --driver=hyperkit -p minikube
+
+        # Download kubectl
+        minikube kubectl -- get po -A
+
+        # Update permissions for .kube and .minikube
+        sudo chown -R $USER $HOME/.kube $HOME/.minikube
+
+        sudo kubectl config use-context minikube
+
+        # Run Minikube Tunnel
```

Comment:
this works in the fluvio ci: 

```
- name: Install Minikube on Linux
        if: startsWith(matrix.os,'ubuntu')
        uses: manusa/actions-setup-minikube@v2.0.1
        with:
          minikube version: "v1.13.1"
          kubernetes version: "v1.19.2"
          github token: ${{ secrets.GITHUB_TOKEN }}
      - name: Install Minikube on Mac
        if: startsWith(matrix.os, 'macOS')
        run:  brew install minikube
      - name: Setup Minikube for Mac
        if: startsWith(matrix.os, 'macOS')
        run: |  
          minikube start --driver=virtualbox
      - name: Test minikube
        run: |
          minikube profile list
      - name: Build
        run: |
          cargo build
      - name: setup context
        run: |
          ./target/debug/fluvio cluster set-minikube-context
```

Reply:
Yes, but it did not work in the github composite action runner, so I needed to change to the the current.

without this step, calling the `fluvio cluster set-minikube context` in `action-install-fluvio-cluster.sh#L40` fails, as checked by the pre-install check called subsequently.

Reply:
hmm. we should create issue to resolve problem them